src/OpenGL/Silk.NET.OpenGL/GL.cs

```diff
@@ -42,186 +98,512 @@ public bool TryGetExtension<T>(out T ext)
+        /// </summary>
+        /// <param name="extension">
+        ///     Standard identifier for the extension.
+        /// </param>
+        /// <returns>
+        ///     <c>True</c> if the extension is present, otherwise <c>False</c>.
+        /// </returns>
         public override bool IsExtensionPresent(string extension)
         {
             _extensions ??= Enumerable.Range(0, GetInteger(GLEnum.NumExtensions))
-                .Select(x => GetString(StringName.Extensions, (uint)x))
+                .Select(x => GetString(StringName.Extensions, (uint) x))
                 .ToList();
 
             return _extensions.Contains("GL_" + (extension.StartsWith("GL_") ? extension.Substring(3) : extension));
         }
 
-        public void ClearColor(Color color)
-        {
-            ClearColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f);
-        }
-
-        public void BlendColor(Color color)
-        {
-            BlendColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f);
-        }
-
-        public void Uniform2(int location, ref Vector2 vector)
-        {
-            Uniform2(location, vector.X, vector.Y);
-        }
-
-        public void Uniform3(int location, ref Vector3 vector)
-        {
-            Uniform3(location, vector.X, vector.Y, vector.Z);
-        }
-
-        public void Uniform4(int location, ref Vector4 vector)
-        {
-            Uniform4(location, vector.X, vector.Y, vector.Z, vector.W);
-        }
-
-        public void Uniform2(int location, Vector2 vector)
-        {
-            Uniform2(location, vector.X, vector.Y);
-        }
-
-        public void Uniform3(int location, Vector3 vector)
-        {
-            Uniform3(location, vector.X, vector.Y, vector.Z);
-        }
-
-        public void Uniform4(int location, Vector4 vector)
-        {
-            Uniform4(location, vector.X, vector.Y, vector.Z, vector.W);
-        }
-
-        public void Uniform4(int location, Quaternion quaternion)
-        {
-            Uniform4(location, quaternion.X, quaternion.Y, quaternion.Z, quaternion.W);
-        }
-
-        public void ProgramUniform2(uint program, int location, ref Vector2 vector)
-        {
-            ProgramUniform2(program, location, vector.X, vector.Y);
-        }
-
-        public void ProgramUniform3(uint program, int location, ref Vector3 vector)
-        {
-            ProgramUniform3(program, location, vector.X, vector.Y, vector.Z);
-        }
-
+        /// <summary>
+        ///     Invokes a call to <c>glClearColor()</c>, setting the clear color of the OpenGL context.
+        /// </summary>
+        /// <param name="color">
+        ///     New clear color for the OpenGL context.
+        /// </param>
+        public void ClearColor(Color color) { ClearColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f); }
```

Comment:
This change goes against the code style for the long line limit of 120 characters and also should be an expression-style body. You can solve this using:
```suggestion
        public void ClearColor(Color color)
            => ClearColor(color.R / 255.0f, color.G / 255.0f, color.B / 255.0f, color.A / 255.0f);
```